src/test/java/taskEnum/MyEnum.java

```diff
@@ -0,0 +1,29 @@
+package taskEnum;
+
+public enum MyEnum {
+    FIRSTHUMAN("Jojo","afrikan", 18),
+    SECONDHUMAN("Ivan","european", 25),
+    THIRDHUMAN("Chi","asian", 71);
+
+    protected String name;
+    protected String humanRace;
+    protected int humanAge;
+
+    MyEnum(String name,String humanRace, int humanAge) {
+        this.name = name;
+        this.humanRace = humanRace;
+        this.humanAge = humanAge;
+    }
+
```

Comment:
add method of getting enum objectS by race :) so it should return either list of array